### ISSUE TITLE
Error with Sourcemaps

### ISSUE THREAD

**Issue by IngwiePhoenix** [babel/babel-loader#190]
```
ERROR in ./app/Support/ControllerExecutor.oj
Module build failed: TypeError: /Users/Ingwie/Work/BIRD3/app/Support/ControllerExecutor.oj: Cannot assign to read only property 'mappings' of {"version":3,"sources":["app/Support/ControllerExecutor.oj"],"names":[],"mappings":";AAAA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA","file":true,"sourceRoot":"/Users/Ingwie/Work/BIRD3"}
```

[OJ](http://github.com/musictheory/oj) has earned ES6 compatibility to some degree, as it now uses Esprima 2.7. However, piping the output of OJ into Babel causes this issue. Thing is, I have no idea why this is the case.

This is where my loader passes away the data it has:
https://github.com/DragonsInn/oj-loader/blob/master/index.js#L83

And this is how OJ passes the source map:
- Instance: https://github.com/musictheory/oj/blob/master/src/modifier.js#L275
- Passing: https://github.com/musictheory/oj/blob/master/src/compiler.js#L297

So basically, they are regular `var`s. How come i get this error, then?...

**Comment by loganfsmyth** [comment]
I'm triaging old issues on babel-loader. Since this has been inactive for ages, I'm going to close it. Feel free to re-open if there's still something to discuss, but I'm assuming at this point it's been too long to address in a useful way.
